types/Pickup/Pickup.d.ts

```diff
@@ -9,19 +9,17 @@ import { IPickupListParameters } from './PickupListParameters';
 /**
  * The Pickup object allows you to schedule a pickup from your carrier from your customer's residence or place of business.
  * Supported carriers include:
```

Comment:
Let's remove this list so we don't have to keep it updated. Point users to the API docs instead.

Reply:
There is a link to the API docs down below in line 27. I will keep this list, doesn't hurt to include them